modules/persistent-storage-local-create-cr.adoc

```diff
@@ -62,7 +62,7 @@ local volumes.
 <4> The file system that is created when the local volume is mounted for the
 first time.
 <5> The path containing a list of local storage devices to choose from.
-<6> Replace this value with your actual local disks filepath to the LocalVolume resource, such as `/dev/xvdg`. PVs are created for these local disks when the provisioner is deployed successfully.
+<6> Replace this value with your actual local file system path to the LocalVolume resource, such as `/etc/motd`. PVs are created for these local file systems when the provisioner is deployed successfully.
```

Comment:
No, `/path/to/device` is correct.
filesystem means when the pv is exported to a container it's a filesystem.

Reply:
> No, `/path/to/device` is correct.
> filesystem means when the pv is exported to a container it's a filesystem.

I didn't change `/path/to/device` in the annotation, just the description to use something else besides `dev/xvdg`, which we use in the block device example.

So if this is still not correct, can you please report why in the new [BZ1835077](https://bugzilla.redhat.com/show_bug.cgi?id=1835077) that was opened by the TAM in response to the original PR? 

He is asking that we update the `Filesystem` example to show a filesystem path, rather than a block device name/path. 

Also, can you suggest another alternative for description <6> that will address the TAM's request? Thanks!

Reply:
Added comment in bug.